src/lib/webbrowser.js

```diff
@@ -0,0 +1,19 @@
+var $builtinmodule = function(name){
+    var mod = {};
+    var inBrowser = (typeof window != "undefined") && (typeof window.navigator != "undefined");
+
+    mod.open = new Sk.builtin.func(function() {
+        Sk.builtin.pyCheckArgsLen("open", arguments.length, 1, 3);
+        var url = arguments[0];
+        if (!(url instanceof Sk.builtin.str))
+            throw new Sk.builtin.TypeError("webbrowser.open expects 'str' for 'url'");
+
+        if (!inBrowser)
+            return false;
+
+        url = url.$jsstr();
+        window.open(url, "_blank");
+        return true;
+    });
```

Comment:
a coupe of things here that might be useful. 

Make sure you're returning javascript booleans inside a python function.
These should be `Sk.builtin.bool.false$` and `Sk.builtin.bool.true$` respectively

Worth giving the anonymous function a name. 
```javascript
mod.open = new Sk.builtin.func(function open() {
```
This way when the user uses introspection they will correctly see the name of the function. 

Style preferences for `if` statements with curly braces `{}`

You can use the the call signature 

```javascript
new Sk.builtin.func(function open(url) {
```